Dockerfile

```diff
@@ -207,6 +210,10 @@ ENTRYPOINT ["buildkitd"]
 
 FROM binaries AS buildkit-darwin
 
+FROM binaries AS buildkit-freebsd
+COPY --link --from=binaries / /usr/bin/
+ENTRYPOINT ["buildkitd"]
+
```

Comment:
Hm.. right, so (see my comment above); this uses `buildkitd` as entrypoint, but if I see the `binaries-freebsd` stage, that doesn't include `buildkitd` ? (so something looks wrong here)


So possibly this stage should be the same as `buildkit-linux`, which is using `buildkit-export` as starting point (not `binaries`)

Reply:
Oh, thanks. Changed `binaries-freebsd` stage to include `buildkitd`.